control-plane/src/button-clicks.js

```diff
@@ -74,58 +74,6 @@ $('#updated').on('change', function () {
     }
 })
 
-$('#filter-jobs-button').on("click", function () {
```

Comment:
Are we moving this block of code for implementing new feature or for cleaning up? If for the later, I'd recommend doing it in a separate issue/pr.

If there's too much work to separate, we can keep it in this pr this time. But we should clearly describe which parts are for cleaning up and which for new feature.